client/test/unit/specs/components/JobCard.spec.js

```diff
@@ -198,10 +199,9 @@ describe.skip('Unit | Component | JobCard.vue', () => {
   });
 
   describe('computed property #staffingNeededSince', () => {
-    // TODO: it works on local or on browser, but fails in CircleCI :-/
-    it.skip('should format the mission staffing_needed_from date (ex : "2017-07-01" => "Juillet 2017")', () => {
+    it('should format the mission staffing_needed_from date (ex : "2017-07-01" => "Juillet 2017")', () => {
       // Given
-      job.project.staffing_needed_from = '2017-07-01';
+      job.activity.staffing_needed_from = '2017-07-01';
 
       // When
       const staffingNeededSince = component.staffingNeededSince;
```

Comment:
pour une assertion plus parlante dans ce test, utilisons : trim()).to.equal('1 juillet 2017'); (l 210)